src/IDeployWizardContext.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "vscode-azureextensionui";
+
+export interface IDeployWizardContext extends IActionContext {
```

Comment:
Can you move this interface into `setAppWizardContextDefault.ts`? Seems excessive to have a file at the root of `src` just for this